### PR TITLE
New Package: Process-in-Process (PiP)

### PR DESCRIPTION
**Pull request by ahori**
details can be found at https://github.com/RIKEN-SysSoft/PiP

### CODE REVIEW

File: var/spack/repos/builtin/packages/process-in-process/package.py [hunk 1]
```diff
@@ -0,0 +1,177 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/process-in-process/package.py]:
```suggestion
# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```

File: var/spack/repos/builtin/packages/process-in-process/package.py [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+             destination='PiP-gdb',
+             when='@3 os=rhel8')
+
+    # PiP testsuite (agnostic with PiP and OS versions)
+    resource(name='PiP-Testsuite',
+             git='https://github.com/RIKEN-SysSoft/PiP-Testsuite.git',
+             destination='PiP-Testsuite')
+
+    def install(self, spec, prefix):
+        "Install Process-in-Process inclduing PiP-glibc, PiP-gdb"
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/process-in-process/package.py]:
```suggestion
        "Install Process-in-Process including PiP-glibc, PiP-gdb"
```

Comment (ahori) [reply var/spack/repos/builtin/packages/process-in-process/package.py]: Oops, thanks.

**Comment by ahori** [comment]
A test was failed. So I will create a new PR.

**Pull request closed without merging**